src/pages/Authentication/Login.tsx

```diff
@@ -0,0 +1,228 @@
+}
+
+function emailCheck(email_address: string) {
+  const email_regex = /^[a-zA-Z0-9._-]+@[a-zA-Z0-9.-]+\.[a-zA-Z]{2,4}$/i;
+
+  if (!email_regex.test(email_address)) {
+    return false;
+  } else {
+    return true;
+  }
+}
+
+const Login = () => {
+  const [email, setEmail] = useState<InputState>({
+    value: "",
+    Message: "",
+    isValid: true,
+  });
+  const [password, setPassword] = useState<InputState>({
+    value: "",
+    type: "password",
+    Message: "",
+    isValid: true,
+  });
+  const [disabled, setDisabled] = useState<boolean>(true);
+
+  const eyesImgRef = useRef<HTMLImageElement | null>(null);
+
+  const handleEmailChange = (e: ChangeEvent<HTMLInputElement>) => {
+    setEmail((prevEmail) => ({
+      ...prevEmail,
+      value: e.target.value,
+    }));
+
+    if (e.target.value.length > 0) {
+      if (email.Message === "이메일을 입력해주세요") {
+        setEmail((prevEmail) => ({
+          ...prevEmail,
+          Message: "",
+        }));
+        e.target.classList.remove("focusout");
+      }
+    }
+  };
+
+  const handlePasswordChange = (e: ChangeEvent<HTMLInputElement>) => {
+    setPassword((prevPassword) => ({
+      ...prevPassword,
+      value: e.target.value,
+    }));
+
+    if (e.target.value.length > 0) {
+      if (password.Message === "비밀번호를 입력해주세요") {
+        setPassword((prevPassword) => ({
+          ...prevPassword,
+          Message: "",
+        }));
+        e.target.classList.remove("focusout");
+      }
+    }
+  };
+
+  const handleEmailBlur = (e: FocusEvent<HTMLInputElement>) => {
+    let emailValue = e.target.value;
+
+    if (e.target.value === "") {
+      setEmail((prevEmail) => ({
+        ...prevEmail,
+        Message: "이메일을 입력해주세요",
+        isValid: true,
+      }));
+      e.target.classList.add("focusout");
+    } else if (!emailCheck(emailValue) && e.target.value !== "") {
+      setEmail((prevEmail) => ({
+        ...prevEmail,
+        Message: "잘못된 이메일 형식입니다",
+        isValid: true,
+      }));
+      e.target.classList.add("focusout");
+    } else {
+      setEmail((prevEmail) => ({
+        ...prevEmail,
+        Message: "",
+        isValid: false,
+      }));
+      e.target.classList.remove("focusout");
+    }
+  };
+
+  const handlePasswordBlur = (e: FocusEvent<HTMLInputElement>) => {
+    if (e.target.value === "") {
+      setPassword((prevPassword) => ({
+        ...prevPassword,
+        Message: "비밀번호를 입력해주세요",
+        isValid: true,
+      }));
+      e.target.classList.add("focusout");
+    } else if (e.target.value.length < 8) {
+      setPassword((prevPassword) => ({
+        ...prevPassword,
+        Message: "비밀번호를 8자 이상 입력해주세요",
+        isValid: true,
+      }));
+      e.target.classList.add("focusout");
+    } else {
+      setPassword((prevPassword) => ({
+        ...prevPassword,
+        Message: "",
+        isValid: false,
+      }));
+      e.target.classList.remove("focusout");
+    }
+  };
+
+  const eyesButtonClick = () => {
```

Comment:
앞서 정의한 함수들이 handleEmailBlur, handlePasswordChange, handleEmailChange 처럼 
- handle이라는 이름의 prefix로 시작
- email, password 등 대상을 지정
- click, change 등 동작 및 상황을 지정
의 구조를 가지고 있으므로 아래와 같이 수정되면 좋을 것 같습니다.
(화면 상 보이는 눈모양 아이콘은 한 개이므로 복수형 부분도 수정하였습니다.)

```suggestion
  const handleEyeButtonClick = () => {
```